src/pinning.js

```diff
@@ -1,6 +1,8 @@
+require('dotenv').config()
 const IPFS = require('ipfs')
 const OrbitDB = require('orbit-db')
 const Pubsub = require('orbit-db-pubsub')
+const timer = require('exectimer')
```

Comment:
Dont' think there is much value to tracking time of openDB right now. Let's remove this to make it simpler.